opentelemetry/src/logs/record.rs

```diff
@@ -10,6 +10,11 @@ pub trait LogRecord {
     {
     }
 
+    /// Sets the `target` of a record
```

Comment:
some note about the need for this as opposed to instrumentation scope would be helpful.
also mention that exporters would use target to override instrumentation scope when present.

Reply:
Done.